config/config_test.go

```diff
@@ -346,6 +346,26 @@ var _ = Describe("Config", func() {
 			})
 		})
 
+		Context("No CF config", func() {
+			BeforeEach(func() {
+				configFileName = "no_cf_config.yml"
+			})
+
+			It("returns an error", func() {
+				Expect(parseErr).To(MatchError(ContainSubstring("Must specify CF")))
+			})
+
+			Context("and CF checks are disabled", func() {
+				BeforeEach(func() {
+					configFileName = "cf_checks_disabled_config.yml"
+				})
+
+				It("succeeds", func() {
+					Expect(parseErr).To(BeNil())
```

Comment:
Just a very minor comment, but it might be more explicit to use `Expect(parseErr).NotTo(HaveOccurred())`